vars/spinInfra.groovy

```diff
@@ -6,16 +6,17 @@ import uk.gov.hmcts.contino.TerraformTagMap
 import uk.gov.hmcts.contino.MetricsPublisher
 import uk.gov.hmcts.pipeline.AKSSubscriptions
 import uk.gov.hmcts.contino.RepositoryUrl
+import java.time.LocalDate
 
-def call(productName, environment, tfPlanOnly, subscription) {
-  call(productName, null, environment, tfPlanOnly, subscription)
+def call(productName, expiresAfter, environment, tfPlanOnly, subscription) {
+  call(productName, null, expiresAfter, environment, tfPlanOnly, subscription)
 }
 
-def call(product, component, environment, tfPlanOnly, subscription) {
-  call(product, component, environment, tfPlanOnly, subscription, "")
+def call(product, component, expiresAfter, environment, tfPlanOnly, subscription) {
+  call(product, component, expiresAfter, environment, tfPlanOnly, subscription, "")
 }
```

Comment:
this is a breaking change you need to add new versions instead of modifying existing ones (if they are used outside of the jenkins library.

It might be better to add a new function that takes a map of parameters instead so we don't have to keep modifying this.

e.g. like this:
https://github.com/hmcts/cnp-jenkins-library/blob/960df1ec6bfcc85051e965ce42e28f67338d2f57/vars/sectionInfraBuild.groovy#L3

or setting defaults like in https://github.com/hmcts/cnp-jenkins-config/blob/master/jobdsl/organisations-beta.groovy#L50-L55